### DISCUSSION TITLE
Why does "open folder" launch Catfish search?

### DISCUSSION THREAD

**Discussion by ColCorn** [cboxdoerfer/fsearch#554] [Q&A]
Running Fsearch (version 0.3.alpha0) under Kubuntu 24.04 LTS and just noticed today that if I highlight one of the files in the search results list, then right-click and then choose "Open Folder" from the contextual menu, Fsearch no longer opens the folder -- instead it launches Catfish search. (I don't think I have ever used Catfish search; not even sure how the software came to be installed on my machine.)

Does anyone know why it's doing this, and how I might restore its original functionality?

Many thanks for a great application. I use it many times every day!

**Comment by ColCorn** [comment]
Update: I can't remember how I heard about this, but to fix this behavior (at least in a system running KDE Plasma), do:
$ gio mime inode/directory org.kde.dolphin.desktop

Similarly, to get Fsearch to open .txt files with Kate rather than Okular (or some other application), do:
$ gio mime text/plain org.kde.kate.desktop

You can also remedy this behavior in Plasma by going into System Settings/File Associations and re-ordering (or even deleting the entries for) the applications assigned to open various types of files.

**Reply by olcdlm** [reply]
I solved the problem by removing the snap installation and reinstalling via the repository and apt installation.
Everything works perfectly like this on my kubuntu system.

**Reply by olcdlm** [reply]
sudo add-apt-repository ppa:christian-boxdoerfer/fsearch-stable
sudo apt update
sudo apt install fsearch

**Comment by chrberger** [comment]
I experienced the same issue on FreeBSD 14.2 with xfce4.

Checking the current mime binding for folders `xdg-mime query default inode/directory` gave `org.xfce.Catfish.desktop` and hence, I corrected it with `xdg-mime default thunar.desktop inode/directory`.

Now, "Open Folder" works again as expected.

**Reply by JamesGMCoder** [reply]
thank you, worked for me too:
`xdg-mime default thunar.desktop inode/directory`
